### PR TITLE
feat(Files): Show files on files browser

### LINKED ISSUE #275: Files: Get root directory from constellation
#### a. Uplink mobile implementation
image

### LINKED ISSUE #322: Files: Format files size as Uplink mobile
### Format files size as Uplink mobile 
image

### PR DESCRIPTION
**Pull request by lgmarchi**
### 1. Get all files on constellation and show on files browser
https://user-images.githubusercontent.com/63157656/202726933-6cbab4e4-24ce-4a4f-9391-040b14fdd35e.mov

a. For now, we are going to work just with files, no folders 
image

### 2. On each 1 second, it listen the file system again and update files list if there is an update
image

### 3. File component 
a. Format your name to be responsive and not overlap other files
image

b. Format file size with one decimal 
image

### Which issue(s) this PR fixes 🔨
- Resolve #322 
- Resolve #275

### CODE REVIEW

**Comment by InfamousVague** [comment]
Looks great, tagging in @dariusc93 and @sdwoodbury for review!

File: src/components/main/files/browser/mod.rs [hunk 1]
```diff
@@ -22,33 +48,21 @@ pub fn FileBrowser(cx: Scope<Props>) -> Element {
...
-                kind: String::from("txt"),
-                size: 0
-            },
-            File {
-                name: String::from("Cache.zip"),
-                state: State::Secondary,
-                kind: String::from("archive/zip"),
-                size: 1
-            }
+            files_sorted.iter().map(|file| {
```
Comment (dariusc93) [inline src/components/main/files/browser/mod.rs]: Do keep in mind that `Directory::get_items` will return an `Item` that may represent either a `File` or `Directory`. There are functions to check to see if its a file or directory such as `Item::is_file`, `Item::is_directory`, or using `Item::item_type` to get a `ItemType` that may return `ItemType::DirectoryItem` or `ItemType::FileItem` and can be resolved to either one by using `Item::get_file` or `Item::get_directory`. 

So I would suggest the following for now. 

```suggestion
            files_sorted.iter().filter(|item| item.item_type() == ItemType::FileItem).map(|file| {
```

**Review by dariusc93 — COMMENTED** [review]
Looks good so far. I did leave a quick comment on one of the lines and will be taking a closer look shortly in the IDE

File: src/components/main/files/browser/mod.rs (hunk 1 above)
Comment (lgmarchi) [reply src/components/main/files/browser/mod.rs]: Perfect my friend!

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**